lib/exercism/user.rb

```diff
@@ -124,11 +124,20 @@ def count_existing_five_a_day
     ActiveRecord::Base.connection.execute(count_existing_five_a_day_sql).field_values("total").first.to_i
   end
 
-
   def five_a_day_exercises
     @exercises_list ||= ActiveRecord::Base.connection.execute(five_a_day_exercises_sql).to_a
   end
 
+  def show_five?
+    if guest? && !onboarded?
+      false
+    elsif five_a_day_exercises.count + count_existing_five_a_day != 5
```

Comment:
How would you describe `five_a_day_exercises.count + count_existing_five_a_day` in English? Should we extract it into a method for the sake of readability?